client/src/main.h

```diff
@@ -50,6 +50,7 @@ enum MicDefaultState {
   MIC_DEFAULT_ALLOW,
   MIC_DEFAULT_DENY
 };
+#define MIC_DEFAULT_MAX (MIC_DEFAULT_DENY + 1)
```

Comment:
This doesn't follow the format we use for other enumerations, this should be an entry in the enum itself

Reply:
We should change the format. Putting `MIC_DEFAULT_MAX` will require an extra `default` branch when switching or `gcc` will issue a warning saying that not all enum elements are matched. I don't think it's a good idea to keep wrestling with the compiler like that. We already do this with [`LG_ROTATE_MAX`](https://github.com/gnif/LookingGlass/blob/fba7c80b2f7fd134a956aaab4d88b8e7b1b4b294/client/include/interface/renderer.h#L68-L70).